spec/core/ics-026-routing-module/README.md

```diff
@@ -511,6 +511,7 @@ function handleChanOpenAck(datagram: ChanOpenAck) {
       datagram.portIdentifier,
       datagram.channelIdentifier,
       datagram.version,
+      datagram.counterpartyChannelIdentifier,
```

Comment:
Add also `datagram.counterpartyVersion` (to be consistent with ibc-go). 

Also, both `counterpartyChannelIdentifier` and `counterpartyVersion` should be part of the `ChanOpenAck` data structure above.

Reply:
Done. I renamed `version` to `counterpartyVersion` (since that's the version being provided - the channel already has its version set on INIT/Try).